### DISCUSSION TITLE
Ender3S1-F4-UBL-20240126.bin on a E3S1 loose the Z value

### DISCUSSION THREAD

**Discussion by kickfliph** [mriscoc/Ender3V2S1#1238] [Q&A]
Hello there,

I just upgraded my E3S1 to the latest version [Ender3S1-F4-UBL-20240126.bin] and I found some problems like the Z offset value going to 0.0 after turning off the printer so I have to calibrate it every time I turn it on., I had the 2.1.3-20230522 and it was working great.

Regards,

**Comment by mriscoc** [comment]
Current version does not save the z-offset automatically, you need to save settings via menu or  using the G-code M500

**Reply by kickfliph** [reply]
Hello there,

Thanks for your reply. 

Got it! other things are not working as before so I rolled back to the stable version posted. It's working fine so thanks for all the support. 

Regards,

**Reply by kickfliph** [reply]
How do I do this from the menu?

**Reply by mriscoc** [reply]
Control or Advanced menu and then Save Settings
